v3/src/components/axis/components/droppable-axis.tsx

```diff
@@ -4,8 +4,8 @@ import React, { CSSProperties } from "react"
 import { createPortal } from "react-dom"
 import {clsx} from "clsx"
 import { AxisPlace } from "../axis-types"
-import { useAxisBounds } from "../hooks/use-axis-bounds"
 import { DropHint } from "../../graph/components/drop-hint"
+import {useGraphLayoutContext} from "../../graph/models/graph-layout"
```

Comment:
I was initially concerned that this was an additional dependency from `axis` to `graph`. Upon closer inspection, however, it appears that `DroppableAxis` is _only_ used by the graph, so I think the entire `DroppableAxis` component should be moved into the `graph` folder.